automation/check-patch.e2e-multus-dynamic-networks-controller-functests.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -xeuE
+
+# This script should be able to execute macvtap-cni
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-macvtap-functests.sh
+
+teardown() {
+    cp $(find . -name "*junit*.xml") $ARTIFACTS || true
+    rm -rf "${TMP_COMPONENT_PATH}"
+    cd ${TMP_PROJECT_PATH}
```

Comment:
it `rm` and then cd the folder to use `make cluster-down`
need to reorder here please (hope it is fine i comment on closed PR if not please let me know)

Reply:
it removes `TMP_COMPONENT_PATH ` then moves to `TMP_PROJECT_PATH` - i.e. `/tmp/cnao/...` . 

Those are not the same.

Does this address your concern ?

Reply:
All good. It would be indeed worrying - I am pretty sure i just copied this from the other tests.

Reply:
btw line 12 there is a typo
and what bit worrying is that in case no junit exists it will pass